sdk/appcenter/src/test/java/com/microsoft/appcenter/ingestion/AppCenterIngestionTest.java

```diff
@@ -175,10 +179,12 @@ public void onBeforeCalling() throws Exception {
         /* Mock instances. */
         URL url = new URL("http://mock/path/file");
         String appSecret = UUIDUtils.randomUUID().toString();
+        String authToken = UUIDUtils.randomUUID().toString();
         String obfuscatedSecret = HttpUtils.hideSecret(appSecret);
+        String obfuscatedToken = HttpUtils.hideSecret(authToken);
```

Comment:
At the end of this method you should check that the log contains the obfuscated token, otherwise this isn't being used.